scripts/mistica-skin-generator/generators/colors/reduce-colors.js

```diff
@@ -0,0 +1,59 @@
+export const MISTICA_COLOR = "MisticaColor";
+export const UI_COLOR = "UIColor";
+export const GRADIENT_COLOR = "MisticaGradient";
+
+export const reduceColors = (brandsWithTokens) =>
+  brandsWithTokens.reduce((acc, brandWithTokens) => {
+    const brandId = brandWithTokens.brand.id;
+    const light = brandWithTokens.tokens.light;
+    Object.entries(light).forEach((entry) => {
```

Comment:
you can use destructuring in function params:
```suggestion
    Object.entries(light).forEach(([colorToken, colorDescription]) => {
```